### ISSUE TITLE
Calculation of AUPRC in GPN-MSA Figure 2b

### ISSUE THREAD

**Issue by yangzhao1230** [songlab-cal/gpn#26]
I'm attempting to reproduce the results shown in Figure 2b, but the AUPRC values I'm calculating seem odd. I've been using the scores provided by your Hugging Face implementation. Could you provide a simple code snippet for replicating the results in Figure 2b?

I have demonstrated my calculation process in a self-contained Colab notebook, which you can access here: [Colab Notebook Link](https://colab.research.google.com/drive/1Gm5uF3GV5YCUJRRrhBJ00aO6lw5YqOYu?usp=sharing). Could you please take a look and let me know if there's anything I'm missing?

**Comment by gonzalobenegas** [comment]
Hello! I believe you just need to flip the sign of the scores. Lower means more deleterious, so scores are anti-correlated with label. Apologies that this is not documented.

BTW, `songlab/cosmic`  is for the upcoming v2 of the manuscript, with slight differences from v1.

**Comment by yangzhao1230** [comment]
Whether or not the labels are flipped, the results are very strange.
![image](https://github.com/songlab-cal/gpn/assets/52462696/716defa8-d9cd-48d9-b22f-d59871c7d8b1)
![image](https://github.com/songlab-cal/gpn/assets/52462696/45ce4508-8bb7-46b8-ac36-76305bc89c4d)

**Comment by yangzhao1230** [comment]
Functions I used to calculate the results are from sklearn
`from sklearn.metrics import precision_recall_curve, auc, average_precision_score`

**Comment by gonzalobenegas** [comment]
See edited notebook: [link](https://colab.research.google.com/drive/1yFVeNiCVpDtASYN58JN-QGT51jD-76cE?usp=sharing)

I flipped the scores instead of the labels.
